src/backend/updates.rs

```diff
@@ -71,19 +78,93 @@ fn post_upgrade() {
 }
 
 async fn post_upgrade_fixtures() {
+    mutate(|state| state.read_only = true);
+    let total_posts = read(|state| state.posts.len());
+
+    // Init all stable structs becasue we do not need the old contents of the stable memory anymore
+    init_stable_structs();
```

Comment:
Let's move it to post-upgrade to make sure that stable structs are initialized after upgrade even if this function traps.